scripts/lib/format-converter.ts

```diff
@@ -101,3 +112,9 @@ export function flatToNested(
   }
   return objs;
 }
+
+function coerceToBoolean(input: string): boolean | string {
+  if (input.toLowerCase() === 'true') return true;
+  if (input.toLowerCase() === 'false') return false;
+  return input;
```

Comment:
do we ever expect this to return the string in a non-error way? If not, then could we make this an error instead?

Reply:
(and if so, should there be a test for it?)

Reply:
The output of this feeds into more robust schema validation with `ajv`, which will fail if the value is a non-convertible boolean.

The goofy thing is that ajv can handle `"true"` (the string) to `true` (the boolean) in its conversion, but it can't handle `"TRUE"`, which is how the csv/google sheets booleans are represented. So I needed some conversion method to basically help ajv along. My first version of this wasn't even this function, it was literally just lowercasing the string (which worked), but I decided that might be more confusing.

It seemed a bit redundant to have our own error handling here, hence the "minimal effort" part of the conversion.

Given that extra context, do you still think we should handle it too or is it okay as is? I can definitely add some of the above context in comments as an alternative.

Reply:
Adding context in comments seems reasonable - I think it's a little close to a code smell for this function to exist with only further validation downstream of the non-convertible boolean, but tradeoff might be fine for this case.